From b12ffb9df7a6a8956b08206c36be70fac78508bd Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Thu, 20 Jul 2023 00:18:59 +0100
Subject: [PATCH] Update warning regex for groff 1.23.0

Part of https://gitlab.com/man-db/man-db/-/issues/25.

* man/check-man: groff 1.23.0 says "cannot break line" rather than
"can't break line"; update regular expression.
---
 man/check-man | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/check-man b/man/check-man
index a58b7f9d..f8985a0d 100755
--- a/man/check-man
+++ b/man/check-man
@@ -31,7 +31,7 @@ errors="$(LC_ALL=C MANWIDTH=80 MAN_KEEP_FORMATTING=1 "$top_builddir/libtool" \
 # Indeed, even for other languages we're going to get "cannot adjust line"
 # if %manpath_config_file% expands to something long. Hmm. We'll just ignore
 # this across the board for now.
-errors="$(echo "$errors" | $EGREP -v "(cannot adjust line|can't break line)")" || true
+errors="$(echo "$errors" | $EGREP -v "(cannot adjust line|(can't|cannot) break line)")" || true
 
 if [ "$code" != 0 ]; then
 	echo "man -E UTF-8 -l $1 failed with exit status $code and error output:"
-- 
GitLab

